fluxquery.go

```diff
@@ -0,0 +1,34 @@
+package kapacitor
+
+import (
+	"sync"
+	"time"
+)
+
+type QueryFlux struct {
+	org     string
+	orgID   string
+	stmt    string
+	queryMu sync.Mutex
+	Now     time.Time
+}
+
+func NewQueryFlux(queryString, org, orgID string) (*QueryFlux, error) {
+	return &QueryFlux{org: org, orgID: orgID, stmt: queryString}, nil
+}
+
+// Deep clone this query
+func (q *QueryFlux) Clone() (*QueryFlux, error) {
+	q.queryMu.Lock()
+	defer q.queryMu.Unlock()
+	return &QueryFlux{
+		stmt:  q.stmt,
+		org:   q.org,
+		orgID: q.orgID,
```

Comment:
Why don't we clone `Now` ?

Reply:
It shouldn't matter as that time never gets used, but you are right, for completeness and readability it is better to also clone the now time.